libs/pilight/core/json.c

```diff
@@ -1436,3 +1436,28 @@ int json_find_string(JsonNode *object, const char *name, char **out) {
 void json_free(void *a) {
 	free(a);
 }
+
+JsonNode *json_vartype(int vartype) {
+	JsonNode *vartype_array = json_mkarray();
+
+	if (0 != (vartype & JSON_NULL)) {
+		json_append_element(vartype_array, json_mkstring("null"));
```

Comment:
Just as you did elsewhere, remove the space between the if and the hook. I also would like to swap the condition:
`(vartype & JSON_NULL) != 0`